configs/README.md

```diff
@@ -28,6 +28,7 @@ Note: `eslint-common.json` does not extends AirBnB ESLint configuration, so it s
 | `eslint-common-react.json`   | React-based projects      | all from `eslint-config-airbnb` and `eslint-common.json` and `eslint-plugin-react`, `eslint-plugin-react-hooks`, `eslint-plugin-jsx-a11y` |
 | `eslint-common-node.json`    | Node-based projects       | all from `eslint-config-airbnb-base` and `eslint-common.json`, `eslint-plugin-node`                                                       |
 | `eslint-common-cypress.json` | Cypress subprojects       | all from `eslint-config-airbnb-base` and `eslint-common.json`, `eslint-plugin-cypress`, `eslint-plugin-chai-friendly`                     |
+| `eslint-common-jest.json`    | Jest subprojects          | `eslint-plugin-jest`                                                                                                                      |
```

Comment:
Sorry, I didn't spot that before, but just realized that for all other configuration files we always extend `airbnb` in one of the flavours - either for React or non-React projects (except `eslint-common.json` which is intended).

I'm wondering shouldn't we do that also for `eslint-common-jest.json`? The only challenge I see is that we use Jest for both - React and non-React stuff, so maybe it won't be possible. How do you see it?

General: The main goal is to use Airbnb eslint rules everywhere in the code as a base.

Reply:
I checked the PRs you created for other repositories and it seems like not only `cloudify-blueprint-topology` is using `airbnb*` in `jest` configs (note that `eslint-common-node.json` and `eslint-common-react.json` contains `airbnb*`).

OK, let's do the following:
1. Keep the configuration as it is right now (do not extend `airbnb*` in `eslint-common-jest`)
2. Update L22 to inform users that `eslint-common-jest.json` should not be used alone, can be just "Note: `eslint-common.json` and `eslint-common-jest` don't extend AirBnB ESLint configuration, so they should not be used alone in the UI projects."
3. In PRs you created - make sure that `airbnb*` is effectively included in Jest part of ESLint configurations (it means that not always it's necessary to add `airbnb*`-related config it in the same section as `eslint-common-jest` is added, sometimes it is already added on a higher level)

Reply:
Thanks for checking. Looked quickly through all the places you listed => All seems good 👍🏻